packages/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django==1.2 
+django==1.4.11 
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2015-2241   
**Check Name:** Django: XSS attack via properties in ModelAdmin.readonly_fields   
**Severity:** MEDIUM   
**Fixed Version:** 1.7.6, 1.8b2   
**Description:** Cross-site scripting (XSS) vulnerability in the contents function in admin/helpers.py in Django before 1.7.6 and 1.8 before 1.8b2 allows remote attackers to inject arbitrary web script or HTML via a model attribute in ModelAdmin.readonly_fields, as demonstrated by a @property.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2015-2241